### PR TITLE
Fix bug in the binomial CDF of event_patternfit

### PR DESCRIPTION
**Pull request by kdund**
_Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible_

## What does the code in this PR do / what does it improve?
The binom_CDF as written evaluates to 0.17 when the AFT is 0-- this gives the wrong p-values. 

## Can you briefly describe how it works?
This PR subtracts away the previous CDF at 0 and scales the remaining CDF to be normalised (returns 1 at AFT=1)

## Can you give a minimal working example (or illustrate with a figure)?
See tests

_Please include the following if applicable:_
  - [x] _Update the docstring(s)_
  - [x] _Update the documentation_
  - [x] _Tests to check the (new) code is working as desired. (works locally)

### _Notes on testing_
 - _Until the automated tests pass, please mark the PR as a draft._
 - _On the XENONnT fork we test with database access, on private forks there is no database access for security considerations._

All _italic_ comments can be removed from this template.

### CODE REVIEW

**Comment by GiovanniVolta** [comment]
Hey @kdund,
thank you for fixing it. It looks good to me:
s1_aft_bug

**Review by GiovanniVolta — APPROVED** [review]

**Review by GiovanniVolta — APPROVED** [review]

File: tests/test_patternfit.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    #PDF is positive:
+    s1top = aftobs*s1tot
+    assert(0. <= binom_pmf(s1top, s1tot, aft))
+    #CDF is positive, between 0 and 1, and 0 at 0 and 1 at s1tot:
+    assert(0. <= binom_cdf(s1top, s1tot, aft))
+    assert(binom_cdf(s1top, s1tot, aft) <=1.)
+    assert(0. == binom_cdf(0., s1tot, aft))
+    assert(1. == binom_cdf(s1tot, s1tot, aft))
+
+
```
Comment (JoranAngevaare) [inline tests/test_patternfit.py:19]:
```suggestion

```

**Review by JoranAngevaare — COMMENTED** [review]
Thanks Knut! Also for writing a test, that's very good practice 👍 

Could you also increment the version of the plugin as the code and therefore results of the computation of the plugin change?

**Comment by kdund** [comment]
Update: Unfortunately I have some things to check wrt how the binomial looks-- the CDF should have a jump @ 0 for some values. 
(Sorry @GiovanniVolta )

**Comment by kdund** [comment]
Update: truncating the CDF is not a complete solution, see #991 for a superseding approach

**Pull request closed without merging**